nullaway/src/main/java/com/uber/nullaway/handlers/StreamNullabilityPropagatorFactory.java

```diff
@@ -0,0 +1,119 @@
+package com.uber.nullaway.handlers;
```

Comment:
License and Uber copyright notice (due to CLA, I think) missing on all these new files after the refactoring :)